src/test/java/lambda/part1/exercise/Exercise3.java

```diff
@@ -0,0 +1,64 @@
+package lambda.part1.exercise;
+
+import com.google.common.collect.FluentIterable;
+import lambda.data.Person;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+@SuppressWarnings({"ConstantConditions", "unused"})
+public class Exercise3 {
+
+    @Test
+    public void sortPersonsByAgeUsingArraysSortExpressionLambda() {
+        Person[] persons = getPersons();
+
+        Arrays.sort(persons, (o1, o2) -> Integer.compare(o1.getAge(), o2.getAge()));
+
+        assertArrayEquals(new Person[]{
+                new Person("Иван", "Мельников", 20),
+                new Person("Николай", "Зимов", 30),
+                new Person("Алексей", "Доренко", 40),
+                new Person("Артем", "Зимов", 45)
+        }, persons);
+    }
+
+    @Test
+    public void sortPersonsByLastNameThenFirstNameUsingArraysSortExpressionLambda() {
+        Person[] persons = getPersons();
+
+        Arrays.sort(persons, (o1, o2) -> {
+            return o1.getLastName().equalsIgnoreCase(o2.getLastName()) ? o1.getFirstName().compareToIgnoreCase(o2.getFirstName())
+                    : o1.getLastName().compareToIgnoreCase(o2.getLastName());
```

Comment:
I guess, if you want to use expression-lambda then use it, not single-line statement-lambda.

Reply:
You are right, thanks